app/controllers/HomeController.scala

```diff
@@ -1,17 +1,32 @@
-package controllers
-
-import javax.inject._
-import play.api._
-import play.api.mvc._
-
-@Singleton
-class HomeController @Inject()(cc: ControllerComponents) extends AbstractController(cc) {
-
-  def index() = Action { implicit request: Request[AnyContent] =>
-    Ok(views.html.index())
-  }
-  
-  def hello(name: String) = Action { implicit request: Request[AnyContent] =>
-    Ok(views.html.hello(name))
-  }
-}
+package controllers
+
+import scala.concurrent.Future
```

Comment:
Is there convention for grouping / ordering of imports?

Reply:
I've been doing scala packages, java packages, everything else sorted in alphabetical order and then application packages. Don't remember where exactly I saw that. I'm fine standardizing on whatever.